components/overflow-group/overflow-group.js

```diff
@@ -15,7 +15,6 @@ import { LocalizeCoreElement } from '../../lang/localize-core-element.js';
 import { offscreenStyles } from '../offscreen/offscreen.js';
 import ResizeObserver from 'resize-observer-polyfill/dist/ResizeObserver.es.js';
 import { RtlMixin } from '../../mixins/rtl-mixin.js';
-import throttle from 'lodash-es/throttle';
```

Comment:
Should I just remove the lodash-es import until we have another use case for it?

Reply:
I think yeah, might as well remove it entirely. We can always re-introduce it later if it's needed.